recipes/lsstdesc.coord/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "LSSTDESC.Coord" %}
+{% set version = "1.0.5" %}
+{% set sha256 = "fca209035123683cc666aa20d01ea5f67b31f370f2f5195af8a7e8db8bda484f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0]|lower }}/{{ name|lower }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - setuptools
+    - pip
+    - numpy >=1.13
+    - future >=0.15
+    - cffi >=1.11.0
+
+  run:
+    - python
+    - numpy >=1.13
```

Comment:
are you compiling against ``numpy``  or is it just a dependency?

Reply:
Just runtime. I will remove the host requirement and remove the version spec as well since that is not in the upstream repo. Good catch!

Reply:
Done.